src/sgame/sg_bot_nav.cpp

```diff
@@ -613,7 +613,7 @@ Global Bot Navigation
 =========================
 */
 
-void BotMoveToGoal( gentity_t *self )
+bool BotMoveToGoal( gentity_t *self )
```

Comment:
While we are here, can we start documenting some of these functions. Especially now that you are adding a return value? It's not clear to me what return true and return false mean.

Reply:
Yes, all the navigation stuff is heavily *UN*documented. As for everything sadly. I'll document this when I'll have fixed the mess.